### PR TITLE
Add 'tier' as short name for NSTemplateTier CRD

### PR DESCRIPTION
**Pull request by alexeykazakov**
This PR adds a short name 'tier' for NSTemplateTier CRD. So we can do `oc get tier`.

Auto-generated by https://github.com/codeready-toolchain/api/pull/41

### CODE REVIEW

File: deploy/crds/toolchain_v1alpha1_nstemplatetier_crd.yaml [hunk 1]
```diff
@@ -10,6 +10,8 @@ spec:
   names:
     kind: NSTemplateTier
     plural: nstemplatetiers
+    shortNames:
```
Comment (xcoulon) [inline deploy/crds/toolchain_v1alpha1_nstemplatetier_crd.yaml]: should we also have `singular: nstemplatetier` to be consistent with the plural name?

Comment (MatousJobanek) [reply deploy/crds/toolchain_v1alpha1_nstemplatetier_crd.yaml]: isn't it set by default - that you can use lowercase `nstemplatetier` without specifying it in `shortNames`?

Comment (alexeykazakov) [reply deploy/crds/toolchain_v1alpha1_nstemplatetier_crd.yaml]: Yes. `nstemplatetier` works OOTB. No need to add it to `shortNames` explicitly.

Comment (xcoulon) [reply deploy/crds/toolchain_v1alpha1_nstemplatetier_crd.yaml]: ok, sorry for the dumb question 🤦‍♂

**Pull request merged**